### ISSUE TITLE
className not being applied to table

### ISSUE THREAD

**Issue by nickgermaine** [carlosrocha/react-data-components#59]
I'm writing this:

    <DataTable
        keys="name"
        className="table table-stripped tickets-table dataTable no-footer"
        columns={[]}
        initialData={[]}
        useShowHideColumns={true}
        initialPageLength={10}
        initialSortBy={{ prop: 'name', order: 'descending' }}
        pageLengthOptions={[10, 25, 50, 100]}
    />
                    
It's rendering this:               

     <div class="container"><div class="row"><div class="col-xs-4"><div><label for="page-menu">Page size:</label><select id="page-menu"><option value="10">10</option><option value="25">25</option><option value="50">50</option><option value="100">100</option></select></div><div><label for="search-field">Search:</label><input type="search" id="search-field" value=""></div></div><div class="col-xs-8"></div></div><table class="table table-bordered"><caption class="sr-only" role="alert" aria-live="polite">Sorted by name: descending order</caption><thead><tr></tr></thead><tbody><tr><td colspan="0" class="text-center">No data</td></tr></tbody></table></div>

Using this:
"react-data-components": "^1.2.0",

**Comment by gavargas22** [comment]
It seems that this is addressed in #14 but it has not been merged you can use that version.

**Comment by DuudeXX8** [comment]
Yup. Classname also wouldn't work for me.Please solve this issue.
